src/Sound.js

```diff
@@ -87,15 +87,17 @@ export default function Sound(props) {
     };
     fetchSound();
   }, [id, freeSound]);
-
   useEffect(() => {
     const unsubscribe = tinykeys(window, {
       Space: (event) => {
-        event.preventDefault();
-        setIsPlaying((i) => !i);
+        if (document.activeElement === document.body) {
```

Comment:
nit: we should probably add some comments here explaining the check we're performing here

Reply:
For sure. I added a couple comments, as you'll see in the incoming commit.